### DISCUSSION TITLE
How do I filter rows with a non-empty array column in the Studio?

### DISCUSSION THREAD

**Discussion by TomasHubelbauer** [supabase/supabase#14092] [Questions] [answered]
Hi, I am wondering if it is possible to filter out rows whose array-type cell value is a non-empty array in the Supabase Studio? I tried using `<> []` and `<> ARRAY()` but this just gives me an empty page with a footer saying there was an error fetching the data. Is this simply not possible?

**Comment by GaryAustin1** [comment, marked as answer]
Just use `{}`  no quotes.  That is postgres string for array.

**Reply by TomasHubelbauer** [reply]
Thanks!
